README.md

```diff
@@ -817,9 +817,29 @@ Then pass the generated `custom_genesis_params` `dict` to the backend's `__init_
 >>> t = EthereumTester(backend=pyevm_backend)
 ```
 
-Overriding genesis state is similar to overriding genesis parameters but requires the consideration of test accounts.
-To override the genesis state of accounts, pass a `state_overrides` `dict` to `PyEVMBackend._generate_genesis_state`,
-and optionally, the number of accounts to create.  
+To override the genesis state of accounts, pass an `overrides` `dict` to `PyEVMBackend._generate_genesis_state`.
+Optionally, provide `num_accounts` and/or a `mnemonic` to control which accounts get overridden. When creating the
+backend, you may also provide a `mnemonic` to set which accounts it uses. This allows you to fund and use your own 
+set of accounts in `EthereumTester`.
+```python
+>>> from eth_tester import PyEVMBackend, EthereumTester
+>>> from eth_utils import to_wei
+>>> from hexbytes import HexBytes
+>>>
+>>> # The following address stems from the mnemonic
+>>> address = '0x1e59ce931B4CFea3fe4B875411e280e173cB7A9C'
+>>> mnemonic = 'test test test test test test test test test test test junk'
+>>>
+>>> genesis_state = PyEVMBackend._generate_genesis_state(
+>>>   overrides={'balance': to_wei(1000000, 'ether')},
+>>>   mnemonic=mnemonic,
+>>>   num_accounts=1
+>>> )
+>>> pyevm_backend = PyEVMBackend(genesis_state=genesis_state, mnemonic=mnemonic)
```

Comment:
I may be misunderstanding something, but I ran through this example locally and it feels a little bit clunky that you have to pass in `mnemonic` here and in the `genesis_state`. Is there a way to simplify and only pass it with `genesis_state`? Or is that a check to make sure the `mnemonic` is the same?

Reply:
The best solution I found is to create a helper factory classmethod `from_mnemonic` that does both... It is tricky because both use the private keys for different purposes and i couldn't really find a way to get rid of one and not the other. I really like the class method approach though! (You will see when I push). Now, it is even simpler to do this and the user only needs to pass the mnemonic once.